File: C#/Primeiros desafios com C#/Mes.cs
// Mês

/*
Leia um valor inteiro entre 1 e 12, inclusive. Correspondente a este valor,
deve ser apresentado como resposta o mês do ano por extenso, em inglês, com a
primeira letra maiúscula.

•	Entrada

A entrada contém um único valor inteiro.

•	Saída

Imprima por extenso o nome do mês correspondente ao número existente na
entrada, com a primeira letra em maiúscula.
*/

using System;

public class Problem {
  public static void Main() {
    int mes = int.Parse(Console.ReadLine());

    switch (mes) {
      case 1:
        Console.WriteLine("January");
        break;
      case 2:
        Console.WriteLine("February");
        break;
      case 3:
        Console.WriteLine("March");
        break;
      case 4:
        Console.WriteLine("April");
        break;
      case 5:
        Console.WriteLine("May");
        break;
      case 6:
        Console.WriteLine("June");
        break;
      case 7:
        Console.WriteLine("July");
        break;
      case 8:
        Console.WriteLine("August");
        break;
      case 9:
        Console.WriteLine("September");
        break;
      case 10:
        Console.WriteLine("October");
        break;
      case 11:
        Console.WriteLine("November");
        break;
      case 12:
        Console.WriteLine("December");
        break;
      default:
        Console.WriteLine("Digite um número válido de 1 a 12");
        break;
    }
    
    Console.ReadLine();
  }
}

File: Java/Primeiros desafios de código com Java/Mes.java
// Mês

/*

*/

import java.io.IOException;
import java.util.Scanner;

public class Problem {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    int month;
    
    month = input.nextInt();
    
    switch (month) {
      case 1:
        System.out.printf("January\n");
        break;
      case 2:
        System.out.printf("February\n");
        break;
      case 3:
        System.out.printf("March\n");
        break;
      case 4:
        System.out.printf("April\n");
        break;
      case 5:
        System.out.printf("May\n");
        break;
      case 6:
        System.out.printf("June\n");
        break;
      case 7:
        System.out.printf("July\n");
        break;
      case 8:
        System.out.printf("August\n");
        break;
      case 9:
        System.out.printf("September\n");
        break;
      case 10:
        System.out.printf("October\n");
        break;
      case 11:
        System.out.printf("November\n");
        break;
      case 12:
        System.out.printf("December\n");
        break;  
      default:
        System.out.printf("Invalid input\n");
    }
  }   
}